assemble/src/main/assemblies/component.xml

```diff
@@ -23,7 +23,7 @@
       <directoryMode>0755</directoryMode>
       <fileMode>0644</fileMode>
       <useProjectArtifact>false</useProjectArtifact>
-      <outputFileNameMapping>${artifact.artifactId}${dashClassifier?}.${artifact.extension}</outputFileNameMapping>
+      <outputFileNameMapping>${artifact.artifactId}-${artifact.version}${dashClassifier?}.${artifact.extension}</outputFileNameMapping>
```

Comment:
This line should just be deleted. This is the default `outputFileNameMapping`.

Reply:
Fixed in 0247ac376dcf123